chains/evm/contracts/pools/USDC/USDCTokenPoolProxy.sol

```diff
@@ -119,11 +155,18 @@ contract USDCTokenPoolProxy is Ownable2StepMsgSender, IPoolV1, ITypeAndVersion {
       revert NoLockOrBurnMechanismSet(lockOrBurnIn.remoteChainSelector);
     }
 
-    // Transfer the tokens to the proper child pool, as this contract is only a proxy and will not perform
-    // the lock/burn itself.
-    i_token.safeTransfer(childPool, lockOrBurnIn.amount);
+    // Transfer the tokens to the correct address, as this contract is only a proxy and will not perform the lock/burn itself.
+    // If using the CCTP verifier, transfer funds to the verifier instead of the child pool.
+    i_token.safeTransfer(
```

Comment:
Let's make this a normal if else with the two statements inside

update: why not just do all of the CCV stuff in the if statement above to avoid the boolean `useCCV` completely